assembl/static2/js/app/components/harvesting/harvestingBox.jsx

```diff
@@ -67,7 +75,10 @@ class DumbHarvestingBox extends React.Component<Object, Props, State> {
       isEditable: false,
       editableExtract: extract ? extract.body : '',
       extractNature: extract && extract.extractNature ? extract.extractNature.split('.')[1] : null,
-      extractAction: extract && extract.extractAction ? extract.extractAction.split('.')[1] : null
+      extractAction: extract && extract.extractAction ? extract.extractAction.split('.')[1] : null,
```

Comment:
are you sure there is always a `.` in `extract.extractAction` ?

Reply:
Hence the 'extract &&' check ? This is not my code. @paolina92 might answer faster to this